skyfield/documentation/positions.rst

```diff
@@ -39,7 +39,8 @@ together with all of the attributes and methods that they support:
 
     Apparent position only
      │
-     └── `altaz(…) <api.html#Position.altaz>`_            →   alt, az, distance
+     └── `altaz(…) <api.html#Position.altaz>`_                 →   alt, az, distance
+     └── `over_location(…) <api.html#Apparent.over_location>`_ →   topos
```

Comment:
I like how you added this in a way that is so symmetric with the rest of the docs! I will have to think about the name a bit, though; I have normally heard this referred to using the “sub” prefix, like the “sublatitude” and “sublongitude” of the satellite, and HORIZONS uses standard phrases like “Sun sub-longitude & sub-latitude” and “Sub-Sun position angle & distance” and “Observer sub-lon & sub-lat” — so I will probably be tempted to switch the language around to match those existing conventions that astronomers are familiar with, but I'll think about it for a few weeks and decide at PyCon. :)